gems/smithy-client/lib/smithy-client/auth/auth.rb

```diff
@@ -54,7 +54,6 @@ def try_load_auth_scheme(auth_option, auth_scheme, identity_providers, failures)
           {
             scheme_id: scheme_id,
             identity: identity_provider.identity,
-            signer: auth_scheme.signer
```

Comment:
Before I forget, I think we will need to check if identity actually returns something here, before returning from this method. It could raise or return nil and this continues.